yew-macro/src/html_tree/html_component.rs

```diff
@@ -151,7 +151,7 @@ impl ToTokens for HtmlComponent {
         };
 
         let key = if let Some(key) = props.key() {
-            quote_spanned! { key.span()=> Some(#key) }
+            quote_spanned! { key.span() => Some(::yew::virtual_dom::Key::from(#key)) }
```

Comment:
Careful @jstarry , there should be no space here before `=>`, cf. https://docs.rs/quote/1.0.7/quote/macro.quote_spanned.html.
I will take care of it.

Reply:
Weird, not sure how that happened! Nice catch